src/libs/KeyboardShortcut/index.js

```diff
@@ -3,8 +3,17 @@ import _ from 'underscore';
 const events = {};
 const keyboardShortcutMap = {};
 
+/**
+ * Return the key-value pair for shortcut keys and translate keys
+ * @returns {Array}
+ */
 function getKeyboardShortcutMap() {
```

Comment:
This is no longer returning a map but an array of objects, so we should rename the function to `getKeyboardShortcuts`